index.html

```diff
@@ -16,6 +16,7 @@
 		</section>
 		<section>
 			<h2>Introduction</h2>
+			<p> this is a test </p>
```

Comment:
This is my comment.

Reply:
Looks like the editing process is working well.